src/renderer/components/MapFilter/ReportView/PDFReport.js

```diff
@@ -103,36 +101,53 @@ const PDFReport = ({
 
 const FeaturePage = ({ observation, getPreset, getMedia }: PageProps) => {
   var view = new ObservationView(observation, getPreset, getMedia)
+  // TODO: move all of these Views into ObservationView
   return (
     <View style={styles.pageContent}>
       <View style={styles.columnLeft}>
-      <Text style={styles.presetName}>{view.preset.name || 'Observation'}</Text>
-      {view.createdAt ? (
-        <Text style={styles.createdAt}>
-          <Text style={styles.createdAtLabel}>Registrado: </Text>
-          <FormattedTime
-            key="time"
-            value={view.createdAt}
-            year="numeric"
-            month="long"
-            day="2-digit"
-          />
+        <Text style={styles.presetName}>
+          {view.preset.name || 'Observation'}
         </Text>
-      ): null}
-      {view.coords ? (
-        <Text style={styles.location}>
-          <Text style={styles.locationLabel}>Ubicación: </Text>
-          <FormattedLocation {...view.coords} />
-        </Text>
-      ): null}
-      <View>
-        {view.note ?
-          view.note.split('\n').map((para, idx) => (
-            <Text key={idx} style={styles.description}>
-              {para}
-            </Text>
-          )): null}
+        {view.createdAt ? (
+          <Text style={styles.createdAt}>
+            <Text style={styles.createdAtLabel}>Registrado: </Text>
+            <FormattedTime
+              key='time'
+              value={view.createdAt}
+              year='numeric'
+              month='long'
+              day='2-digit'
+            />
+          </Text>
+        ) : null}
+        {view.coords ? (
+          <Text style={styles.location}>
+            <Text style={styles.locationLabel}>Ubicación: </Text>
+            <FormattedLocation {...view.coords} />
+          </Text>
+        ) : null}
+        <View>
+          {view.note
+            ? view.note.split('\n').map((para, idx) => (
+                <Text key={idx} style={styles.description}>
+                  {para}
+                </Text>
+              ))
+            : null}
+        </View>
+        {view.fields.length
+          ? fields(view)
+          : null
+        }
       </View>
+      <ObservationRHS observationView={view} />
+    </View>
+  )
+}
+
+function fields (view: ObservationView) {
+  return (
```

Comment:
this fixes the bug where Detalles would be shown no matter what, even if no fields.

We need to make this code better tho! This was a quick hack